### PR TITLE
update service update in docs

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**- What I did**
1. change service update url in api 1.24 doc
2. add api change in api 1.25

related to issue https://github.com/docker/docker/issues/29597

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Can you check if the same needs to be done for the `DELETE` and `inspect` endpoints?

**Comment by allencloud** [comment]
No. No change needed for endpoint `DELETE` and `inspect`. I checked that.

File: docs/api/v1.24.md [hunk 1]
```diff
@@ -4823,9 +4823,9 @@ Return information on the service `id`.
 
 #### Update a service
 
-`POST /services/(id or name)/update`
+`POST /services/(id)/update`
 
-Update a service. When using this endpoint to create a service using a
+Update a service with full ID. When using this endpoint to create a service using a
```
Comment (thaJeztah) [inline docs/api/v1.24.md]: not sure this is needed `id` may be sufficient, wdyt?

**Comment by thaJeztah** [comment]
ok, thanks for double checking

File: docs/api/v1.24.md (hunk 1 above)
Comment (allencloud) [inline docs/api/v1.24.md]: I insist that we should have "full ID". Here is my reason that in this endpoint we can not use ID prefix to update a service. While other endpoints like 
```
DELETE /services/(id or name)
```
We can use prefix ID to delete a service.

So, there is still some difference.

Comment (thaJeztah) [inline docs/api/v1.24.md]: But we don't specify that a prefix ID can be used anywhere in this document. Without mentioning that, a full ID should be assumed here.

Comment (allencloud) [inline docs/api/v1.24.md]: OK, I think from this point removing this part is reasonable. Updated now.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**